include/openssl/err.h.in

```diff
@@ -56,6 +56,7 @@ extern "C" {
 #  define ERR_NUM_ERRORS  16
 struct err_state_st {
     int err_flags[ERR_NUM_ERRORS];
+    int err_marks[ERR_NUM_ERRORS];
```

Comment:
Either here, or in the source, there should be a comment explaining how this works; I would have thought, e.g., that err_marks is its own stack but that seems to not be the case.